src/EntryPoints/NetworkFunction.php

```diff
@@ -5,18 +5,31 @@
 namespace MediaWiki\Extension\Network\EntryPoints;
 
 use MediaWiki\Extension\Network\Extension;
+use MediaWiki\Extension\Network\NetworkFunction\NetworkConfig;
 use MediaWiki\Extension\Network\NetworkFunction\NetworkPresenter;
 use MediaWiki\Extension\Network\NetworkFunction\NetworkUseCase;
 use MediaWiki\Extension\Network\NetworkFunction\RequestModel;
 use Parser;
 
 class NetworkFunction {
 
+	/**
+	 * @var NetworkConfig
+	 */
```

Comment:
This adds little or no info, depending on the used tools. I'd omit it. Or better yet, use PHP 7.4+. All those comments for the fields in `NetworkConfig` are painful to look at. If it was me adding this code I'd bump the extensions needed PHP version to 7.4, unless there is some strong reason to not do so.

Reply:
I'd rather not push the PHP version past what is required by MediaWiki, but I agree this will be much nicer in PHP 7.4.